src/libs/antares/study/parts/hydro/series.cpp

```diff
@@ -62,6 +62,13 @@ unsigned int EqualizeTSsize(Matrix<double, int32_t>& TScollection1,
 {
     PairOfIntegers pairOfTSsizes(TScollection1.width, TScollection2.width);
 
+    if (pairOfTSsizes.bothZero())
+    {
+        TScollection1.reset(1, HOURS_PER_YEAR);
+        TScollection2.reset(1, HOURS_PER_YEAR);
+        return 1;
```

Comment:
Storage matrix has 365 height (DAYS_PER_YEAR), so we need somehow to parse heights of the matrices.

Reply:
ok, done.